src/cpp/src/visual_language/pipeline.cpp

```diff
@@ -277,7 +277,15 @@ class ov::genai::VLMPipeline::VLMPipelineImpl {
         if (config_map.end() != image) {
             rgbs = {image->second.as<ov::Tensor>()};
         } if (config_map.end() != images) {
-            rgbs = images->second.as<std::vector<ov::Tensor>>();
```

Comment:
alternatively, we can assert that in case of `images` we have `images->second.is<std::vector<ov::Tensor>>()`, because it was initial intention that `images` are for `vector` and `image` for a single tensor.

Reply:
Yes, but in python bindings we already allow passing single image in `images` parameter. So it looks misaligned if sometimes it work, sometimes not. 
https://github.com/openvinotoolkit/openvino.genai/blob/ed02aba9f912f1b2ab0d4e1ccaf78a8abdc534fc/src/python/py_vlm_pipeline.cpp#L182

So maybe rename `images`->`image` in this binding?

Reply:
> So maybe rename `images`->`image` in this binding?

I suppose it should be from the very beginning.. maybe a copy paste issue.

Reply:
`images` is a more generic term. phi3_v also calls it `images`: https://huggingface.co/microsoft/Phi-3.5-vision-instruct/blob/main/processing_phi3_v.py#L322